config/environments/test.rb

```diff
@@ -39,34 +39,38 @@
   # and recreated between test runs. Don't rely on the data there!
   config.cache_classes = true
 
-  # Configure static asset server for tests with Cache-Control for performance
-  config.serve_static_assets = true
-  config.static_cache_control = 'public, max-age=3600'
+  # Do not eager load code on boot. This avoids loading your whole application
+  # just for the purpose of running a single test. If you are using a tool that
+  # preloads Rails for running tests, you may have to set it to true.
+  config.eager_load = false
 
-  # Log error messages when you accidentally call methods on nil
-  config.whiny_nils = true
+  # Disable concurrency to avoid race conditions when lazy-loading classes.
+  # See: https://robots.thoughtbot.com/how-to-fix-circular-dependency-errors-in-rails-integration-tests
+  # TODO: remove on upgrade to Rails 4.2.
+  config.allow_concurrency = false
 
-  # Show full error reports and disable caching
+  # Configure static asset server for tests with Cache-Control for performance.
+  config.serve_static_assets  = true
+  config.static_cache_control = "public, max-age=3600"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.